deploy/008-configure-acm-commands-aggregator.ts

```diff
@@ -257,6 +270,24 @@ const grantPermissions: Permissions = {
     ...getIRMPermissions(),
     ...getOmniChainExecutorOwnerPermissions(OPBNBTESTNET_OMNICHAIN_EXECUTOR_OWNER, OPBNBTESTNET_GUARDIAN),
   ],
+  basesepolia: [
+    ...getResilientOraclePermissions(BASESEPOLIA_RESILIENT_ORACLE),
+    ...getChainlinkOraclePermissions(BASESEPOLIA_CHAINLINK_ORACLE),
+    ...getRedstoneOraclePermissions(BASESEPOLIA_REDSTONE_ORACLE),
+    ...getBoundValidatorPermissions(BASESEPOLIA_BOUND_VALIDATOR),
+    ...getXVSPermissions(BASESEPOLIA_XVS),
+    ...getXVSBridgeAdminPermissions(BASESEPOLIA_XVS_BRIDGE_ADMIN),
+    ...getXVSVaultPermissions(BASESEPOLIA_XVS_VAULT_PROXY),
+    ...getPoolRegistryPermissions(BASESEPOLIA_POOL_REGISTRY),
+    ...getPrimePermissions(BASESEPOLIA_PRIME),
+    ...getPrimeLiquidityProviderPermissions(BASESEPOLIA_PLP),
+    ...getProtocolShareReservePermissions(BASESEPOLIA_PSR),
+    ...getComptrollerPermissions(),
+    ...getVTokenPermissions(),
+    ...getRewardDistributorPermissionsTimebased(),
+    ...getIRMPermissions(),
+    ...getOmniChainExecutorOwnerPermissions(BASESEPOLIA_OMNICHAIN_EXECUTOR_OWNER, BASESEPOLIA_GUARDIAN),
```

Comment:
I believe we don't need this here. We've addressed it in the past three testnets to ensure compatibility and alignment with mainnet permissions, but we don't need them anymore. 

I think we should move `getOmniChainExecutorOwnerPermissions` to this file only as we will not need them in future.